config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml

```diff
@@ -275,3 +275,113 @@ periodics:
     testgrid-tab-name: aks-engine-azure-windows-master-2004-containerd-serial-slow
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs serial Windows tests on a Kubernetes cluster running containerd provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud
+- interval: 24h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-20h2-containerd
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  labels:
+    preset-service-account: "true"
+    preset-azure-windows: "true"
+    preset-windows-repo-list-2004: "true"
+    preset-azure-cred: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-windows-private-registry-cred: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210312-67f589a-master
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
+      - --aksengine-creds=$(AZURE_CREDENTIALS)
+      - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+      - --aksengine-public-key=$(K8S_SSH_PUBLIC_KEY_PATH)
+      - --aksengine-private-key=$(K8S_SSH_PRIVATE_KEY_PATH)
+      - --aksengine-winZipBuildScript=$(WIN_BUILD)
+      - --aksengine-orchestratorRelease=1.20
```

Comment:
should this be at 1.21 now?

Reply:
I'm not sure actually... I thought that didn't really have any impact for upstream PR jobs.

Reply:
It might have an impact since the value of this flag will eventually pass to aks-engine API model, which might cause aks-engine to set the wrong defaults. I recently merged a PR that contains a script to auto-generate yaml files for each release (https://github.com/chewong/test-infra/blob/563d558f4fef19784ede3857c71ee7163a3ef918/config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh) so we can minimize copy-pasting errors and unify most parameters. I am considering to do the same for Windows jobs.